tensorboard/default.py

```diff
@@ -30,6 +30,7 @@
 
 import logging
 import os
+import pkg_resources
```

Comment:
This is technically a third-party dependency, and so (a) should be in a
separate imports section, and (b) needs to be expressed in Bazel. The
second point matters for the Google import process.

Reply:
Got the Bazel dep, but the imports should still be:

```python
import logging
import os

import pkg_resources

from tensorboard.compat import tf
from tensorboard.plugins import base_plugin
```